content/03.results.md

```diff
@@ -246,3 +246,28 @@ If the submitter provided cell types, the submitter annotations are compared to
 A summary of this comparison is included in the cell type report along with a table summarizing the submitter cell type annotations and a UMAP plot where each cell is colored by the submitter annotation.
 The Jaccard similarity index is calculated for all pairs of cell type labels in submitter annotations and `SingleR` annotations and in submitter annotations and `CellAssign` annotations.
 The results from both comparisons are displayed in a stacked heatmap available in the report, an example of which is shown in Figure {@fig:figS5}B.
+
+## Analysis of bulk RNA-seq
+
+Several projects in the ScPCA Portal quantified many samples' expression both at the bulk and single-cell level.
+Previous research has suggested that, compared to bulk RNA-seq, single-cell RNA-seq may fail to capture certain cell types [@doi:10.1038/s41587-020-0465-8] due, for example, to technical aspects of library preparation.
+We therefore sought to ask whether we could identify differences in cell type composition between these two modalities.
+For this analysis, we specifically focused on ScPCA projects with solid tumors, considering only non-multiplexed samples with both sequencing modalities and excluding low-quality single-cell libraries.
+In total, we analyzed 97 samples across five projects: `SCPCP00001`, `SCPCP000002`, `SCPCP000006`, `SCPCP000009`, and `SCPCP000017`.
+Projects `SCPCP000001` and `SCPCP000002` comprise high- and low-grade gliomas, respectively, and were sequenced at the bulk and single-cell levels, and `SCPCP000006`, `SCPCP000009`, and `SCPCP000017` comprise Wilms tumors, CNS tumors, and osteosarcomas, respectively, and were sequenced at the bulk and single-nuclei levels.
+As described in Materials and Methods, we derived pseudobulk expression matrices from the single-nuclei libraries, and we compared their expression to bulk using a series of linear models (one per ScPCA project) predicting bulk from pseudobulk expression, with a random effect controlling for sample (Figure {@fig:fig6}A, Figure {@fig:figS8}A).
+We observed an expected positive relationship between bulk and pseudobulk expression across all projects.
+
+We next performed an overrepresentation analysis to test if cell type marker genes for our consensus cell types were enriched in either modality, which may in turn indicate that certain cell types were more prevalently detected by either modality.
+To this end, we calculated the per-gene median of each project's model residuals and identified outliers, where positive outliers represent genes that are enriched in bulk RNA-seq, and negative outliers represent genes that are enriched in single-cell/nuclei RNA-seq.
```

Comment:
I'm not sure "enriched" is the right word here. Positive outliers are genes with higher expression in bulk RNA-seq than predicted based on the pseudobulk expression, and negative outliers are genes with lower expression in bulk RNA-seq than predicted on the pseudobulk expression.

Reply:
> I'm not sure "enriched" is the right word here

I agree very much, in particular since it implies a different type of analysis to, but I wasn't landing on different verbiage. Will rework with some of this suggested phrasing